docs/roleAndPermission.docs.yaml

```diff
@@ -0,0 +1,134 @@
+paths:
```

Comment:
Refactor this documentation to reflect the actual routes. Take a closer look. This documentation is out of date with the routes files. Also, you can rename this file to `role.docs.yaml` to maintain consistency.